BEIMA.Client/cypress/integration/ABeginningTest.spec.js

```diff
@@ -0,0 +1,18 @@
+/// <reference types="cypress" />
+
+/// The purpose of this test is to login to BEIMA before all the other tests are ran
+/// to ensure that navigating through BEIMA can still happen.
+/// The test database must have a user with this username, but the tester needs to replace the empty
+/// password with a valid one.
+describe("Login to BEIMA", () => {
+  it('Login to BEIMA', () => {
+    cy.visit('http://localhost:3000/')
+    cy.get('[id=username]').scrollIntoView().clear().type("testuser")
```

Comment:
We could use our default user generated by the mongo connector here instead?

Reply:
We could, but that would assume the developer running this has no users in the database. And if that default login gets changed then this would need to be changed as well